src/main/java/org/wildfly/bot/PullRequestFormatProcessor.java

```diff
@@ -89,7 +93,8 @@ void pullRequestFormatCheck(
             String sha = pullRequest.getHead().getSha();
             for (GHCommitStatus commitStatus : pullRequest.getRepository().listCommitStatuses(sha)) {
                 // only update format check, if it was created
-                if (commitStatus.getContext().equals(CHECK_NAME)) {
+                if (commitStatus.getContext().equals(CHECK_NAME)
+                        && commitStatus.getCreator().getLogin().equals(appNameProvider.getBotName())) {
```

Comment:
```suggestion
                if (CHECK_NAME.equals(commitStatus.getContext())
                        && commitStatus.getCreator().getLogin().equals(appNameProvider.getBotName())) {
```
This is a tad bit safer against NPE, even tho it might not occur